### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by jaller94**
All of these seem safe and I tested my Slack bridge a bit with the new dependencies installed.
`yargs` has breaking changes for invalid configs that don't affect us, because the repo has full control over the config and doesn't allow user input.

### CODE REVIEW

File: changelog.d/484.misc [hunk 1]
```diff
@@ -0,0 +1 @@
+Upgrade dependencies with no affecting breaking changes
```
Comment (Half-Shot) [inline changelog.d/484.misc:1]:
```suggestion
Upgrade dependencies
```

I wouldn't include the breaking changes line, we are expected to uphold that promise implicitly unless we bump a major version :)

**Review by Half-Shot — APPROVED** [review]
Otherwise, good to go!

File: changelog.d/484.misc (hunk 1 above)
Comment (jaller94) [reply changelog.d/484.misc:1]: ok

Comment (jaller94) [reply changelog.d/484.misc:1]: Missed to change this before merging:
Changed in 944033902033c9f7a516afc9aa5b34cfeb89e640.

**Pull request merged**